source/faq/questions/using-cypress-faq.md

```diff
@@ -311,35 +311,25 @@ It is possible to upload files in your application but it's different based on h
 
 ## {% fa fa-angle-right %} What is the projectId for?
 
-Once you {% url "set up your tests to record" dashboard-service#Recorded-runs %}, we generate a unique `projectId` for your project and automatically insert it into your `cypress.json` file.
-
-***The `projectId` is a 6 character string in your cypress.json:***
+The `projectId` is a 6 character string that helps identify your project once you've {% url "set up your tests to record" dashboard-service#Recorded-runs %}. It's generated by Cypress and typically is found in your {% url `cypress.json` configuration %} file.
```

Comment:
> This is a slight expansion to the scope of the original ticket.

I have tried to preserve the answer the FAQ question while also addressing (removing) the duplication of content in this section and what's already documented in the _Dashboard Services_ page.

Someone reading the FAQ should still be able to grok what the `projectId` is from this smaller answer. The answer directs them to the fuller documentation if they wanna dig in.

<img width="810" alt="from-faq" src="https://user-images.githubusercontent.com/1855109/44626426-fb46e400-a913-11e8-8ff5-c3080a5fd276.png">